projects/Scraping Medium Articles/scraping_medium.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+import requests
+import re
+from bs4 import BeautifulSoup
+
+# switching to current running python files directory
+os.chdir('\\'.join(__file__.split('/')[:-1]))
+
+# function to get the html of the page
+def get_page():
+	url = input('Enter url of a medium article: ')
+	# handling possible error
+	if (url[:18] != 'https://medium.com') and (url[:17] != 'http://medium.com'):
```

Comment:
instead of if u can use regex pattern matching
```suggestion
	if not re.match(r'https?://medium.com/',url):
```